proto/streamlit/proto/Block.proto

```diff
@@ -44,7 +44,8 @@ message Block {
 
   message Expandable {
     string label = 1;
-    bool expanded = 2;
+    optional bool expanded = 2;
```

Comment:
is the addition of the `optional` keyword needed here? fields in proto3 are always technically optional, so I'm not sure if this makes any real difference (+ I'm not entirely sure if it can somehow affect backwards-compatibility-related things)

Reply:
While properties are indeed `optional` as default, they are not `nullable` (as far as I understand this). For the `update` functionality, we need to differentiate between `expanded` being provided (`False` or `True`) or it being absence (== null) in which case we don't modify the expanded state in the frontend.